module-2/lab-df-calculation-and-transformation/your-code/challenge-3.ipynb

```diff
@@ -44,7 +44,7 @@
   },
```

Comment:
- Great job selecting unique customers!! 
- Sub Problem 2 - Setting Expectations. How to select customers whose aggregated amount_spent is in a given quantile range?: At the beginning of the exercise you are requested to get the 75 and 95 percentiles instead of 90 percentile. You should have done the same as you have done for your 90 percentile.
- Sub Problem 3 - Collecting Information: you should do with a conditional similar to the following: 
a) customers with amount spent below 75 percentile would be labeled as normal
b) customers with amount spent between 75 and 95 percentile would be labeled as preferred 
c) customers with amount spent above 95 percentile would be labeled as VIP
You can reduce various lines of code there. Let's see in the afternoon!!